tests/Drupal/Commenting/InlineVariableCommentUnitTest.inc

```diff
@@ -16,5 +16,9 @@ function lorem() {
   // @var $node \Drupal\node\NodeInterface
   # @var \Drupal\node\NodeInterface $node
   # @var $node \Drupal\node\NodeInterface
+  // /** @var \Drupal\node\NodeInterface $node */
+  // /* @var \Drupal\node\NodeInterface $node */
+  # /* @var \Drupal\node\NodeInterface $node */
+  # /* @var \Drupal\node\NodeInterface $node */
```

Comment:
Is there a reason for this repeated line? Maybe there is a something subtle that I am missing?

Reply:
Good catch, forgot to add an extra `*` to the second line.